Week5/app/src/main/java/com/isabelledionisius/mymovieapp/MovieRoomDatabase.kt

```diff
@@ -0,0 +1,43 @@
+package com.isabelledionisius.mymovieapp
+
+import android.content.Context
+import androidx.room.Database
+import androidx.room.Room
+import androidx.room.RoomDatabase
+
+
+// Annotates class to be a Room Database with a table (entity) of the Word class
+@Database(entities = arrayOf(MovieDataList::class), version = 1, exportSchema = false)
+public abstract class MovieRoomDatabase:RoomDatabase() {
+
+
+    abstract fun wordDao(): DaoMovie
+
+    companion object {
+        // Singleton prevents multiple instances of database opening at the
+        // same time.
+        @Volatile
+        private var INSTANCE: MovieRoomDatabase? = null
+
+        fun getDatabase(context: Context): MovieRoomDatabase {
```

Comment:
You can avoid the entire synchronized thing and just build this within the App.kt file

Reply:
One question here: I don't have an App.kt file. I saw that Luca has one in the solution that he shared with us. In general: What is the purpose of this App.kt file?